### ISSUE TITLE
Limbo hangs on certain devices with error "Connecting to VM Console"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#4]
```
What steps will reproduce the problem?
1. Install Limbo on an ASUS tf101 device.
2. Boot DSL iso image.
3. VM shows as Running status but hangs while "Connecting to VM Console".

What is the expected output? What do you see instead?
VM should boot normally.


Please use labels and text to provide additional information.


```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 10 Sep 2012 at 11:15

**Comment by GoogleCodeExporter** [comment]
```
Issue is caused by Dalvik VM which is blocking SIGUSR1 signals to io threads. 
This is now fixed by replacing with SIGUSR2 in version 0.9.1b-2.
```

Original comment by `ideal2i...@gmail.com` on 10 Sep 2012 at 11:19
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Keeping issue open till release of 0.9.1b-2.
```

Original comment by `ideal2i...@gmail.com` on 10 Sep 2012 at 11:21
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 0.9.1b-2.
```

Original comment by `ideal2i...@gmail.com` on 2 Oct 2012 at 8:00
- Changed state: **Fixed**
